integration-tests/integration/statements/not-submit-statement.spec.js

```diff
@@ -0,0 +1,49 @@
+const YourStatementsPage = require('../../pages/yourStatements/yourStatementsPage')
+const WriteYourStatementPage = require('../../pages/yourStatements/writeYourStatementPage')
+const { ReportStatus } = require('../../../server/config/types')
+const CheckYourStatementPage = require('../../pages/yourStatements/checkYourStatementPage')
+
+context('Submit statement', () => {
```

Comment:
think this wasn't updated.. i'd have been tempted to have just added it as a test to the submit statement spec as it's not really worthy of its own "use case"

Reply:
I did think of putting it in the submit-statement for the same reason that i's such a small test. But then I decided against because this test isn't about submitting a statement.